test/Java9andUp/src/org/openj9/test/access/staticAccessChecks/DenyAccess.java

```diff
@@ -56,7 +56,7 @@ public void testAccessJava9Package() {
 			threwIAE = true;
 		}
 		
-		Assert.assertTrue(threwIAE, "Expected IllegalAccessException not thrown");
+		Assert.assertFalse(threwIAE, "Expected IllegalAccessException not thrown");
```

Comment:
Shouldn't this test be checking a package that isn't exported? sun.misc is an exported package from the jdk.unsupported module.

Reply:
@pshipton Yes the original changes are correct, I just had to modify it to catch Error instead of the Exception